src/laser/Laser.H

```diff
@@ -22,6 +22,7 @@ public:
     std::string m_name {""};
     amrex::Real m_a0 {0.}; /**< Laser peak normalized amplitude */
     amrex::Real m_w0 {0.}; /**< Laser waist */
+    amrex::Real m_propagation_angle_yz {0.};/**< Propagation angle of the pulse in the yz plane (0 is the along the z axis) */
```

Comment:
Unless there is a very specific reason, we are keeping the number of characters per line below 100.
See https://hipace.readthedocs.io/en/latest/contributing/contributing.html for the main conventions we are using (the page is a bit old, but hopefully still largely relevant)
```suggestion
    /** Propagation angle of the pulse in the yz plane (0 is the along the z axis) */
    amrex::Real m_propagation_angle_yz {0.};
```